js/qtrade.js

```diff
@@ -409,32 +410,29 @@ module.exports = class qtrade extends Exchange {
         const marketId = this.safeString (ticker, 'id_hr');
         const symbol = this.safeSymbol (marketId, market, '_');
         const timestamp = this.safeIntegerProduct (ticker, 'last_change', 0.001);
-        const previous = this.safeNumber (ticker, 'day_open');
-        const last = this.safeNumber (ticker, 'last');
-        const day_change = this.safeNumber (ticker, 'day_change');
+        const previous = this.safeString (ticker, 'day_open');
+        const last = this.safeString (ticker, 'last');
+        const day_change = this.safeString (ticker, 'day_change');
         let percentage = undefined;
         let change = undefined;
-        const average = this.safeNumber (ticker, 'day_avg_price');
-        if (day_change !== undefined) {
-            percentage = day_change * 100;
-            if (previous !== undefined) {
-                change = day_change * previous;
-            }
+        const average = this.safeString (ticker, 'day_avg_price');
+        percentage = Precise.stringMul (day_change, '100');
+        if (previous !== undefined) {
+            change = Precise.stringMul (day_change, percentage);
```

Comment:
This is wrong, we had `previous` here before, now it's `percentage`.
Let's remove lines 420-422 from this PR cause it's done inside safeTicker.
Also let's make sure that it builds properly.